docs/spark/how-to-guides/debug.md

```diff
@@ -30,13 +30,40 @@ When you run the command, you see the following output:
 ***********************************************************************
 ```
 
-In this debug mode, `DotnetRunner` does not launch the .NET application, but it waits for it to connect. Leave this command prompt window open.
+In this debug mode, `DotnetRunner` does not launch the .NET application, but waits for it to connect. Leave this command prompt window open.
 
-Now you can run your .NET application with any debugger to debug your application.
+Now you can start your .NET application with a C# debugger ([Visual Studio Debugger for Windows/macOS](https://visualstudio.microsoft.com/vs/) or [C# Debugger Extension in Visual Code](https://code.visualstudio.com/Docs/editor/debugging)) to debug your application.
 
-## Debug Scala code
+## Debugging User Defined Function (UDF)
```

Comment:
Not sure if "User Defined Function" should be lowercase or not (because it's used as an acronym). But the "ing" should be removed.


```suggestion
## Debug User Defined Function (UDF)
```